primeqa/ir/README.md

```diff
@@ -150,12 +150,12 @@ Using a model trained as described [here](https://github.com/primeqa/primeqa/tre
 python primeqa/ir/run_ir.py \
     --engine_type DPR \
     --do_index \
-    --dpr_ctx_encoder_path <context_encoder_model> \
+    --ctx_encoder_name_or_path <context_encoder_model> \
     --embed <part_number>of<parts_total> \
```

Comment:
This arg too is very DPR specific and does it need to be exposed?

Reply:
Left alone as after a separate discussion.